scripts/travis/build-all-in-one-image.sh

```diff
@@ -34,18 +42,54 @@ upload_to_docker() {
     echo 'skip docker upload for PR'
     exit 0
   fi
-  export REPO=$1
-  bash ./scripts/travis/upload-to-docker.sh
+
+  REPO=${1}
+  CMD_ROOT=${1}
+
+  unset major minor patch
```

Comment:
why has been this copied from `upload-to-docker.sh`? Could we just keep it there?

Reply:
with buildx, we could not use tag directly.

`docker tag` just tag local image for local arch, when we push that tags, it will break the oci image with multi arch.

Reply:
ok, but  there's a lot being copied here, and would have to be repeated for other Docker images.
Can we not either parameterize `upload-to-docker.sh` or refactor to avoid repetition?

Reply:
I am fine to go with this and see how it works and refactor other images in a second PR